### DISCUSSION TITLE
An example of an addon.

### DISCUSSION THREAD

**Discussion by sannelo** [fnuecke/oc2#117] [Q&A] [answered]
Can you provide an example implementation of addons?I'm new to this stuff and couldn't find any documentation/instructions for this. Thanks in advance!

**Comment by fnuecke** [comment, marked as answer]
I'm assuming you have some experience in modding Minecraft. If not, I suggest getting your feet wet with that, first.

There's a brief overview with pointers [in the project readme](https://github.com/fnuecke/oc2#development-documentation). The [readme in the API package](https://github.com/fnuecke/oc2/tree/1.18-forge/src/main/java/li/cil/oc2/api) goes into more depth, with some code examples.
